flow/Makefile

```diff
@@ -98,20 +98,40 @@ default: finish
 #
 # ==============================================================================
 
-# Include design and platform configuration
-include $(DESIGN_CONFIG)
+#-------------------------------------------------------------------------------
+# Default target when invoking without specific target.
+.DEFAULT_GOAL := finish
 
-PUBLIC=nangate45 sky130hd sky130hs asap7
+#-------------------------------------------------------------------------------
+# Proper way to initiate SHELL for make
+SHELL          = /bin/bash
+.SHELLFLAGS    = -o pipefail -c
 
-ifneq ($(wildcard $(PLATFORM_BASE)/$(PLATFORM)),)
-  export PLATFORM_DIR = $(PLATFORM_BASE)/$(PLATFORM)
-else ifneq ($(findstring $(PLATFORM),$(PUBLIC)),)
-  export PLATFORM_DIR = ./platforms/$(PLATFORM)
-else ifneq ($(wildcard ../../$(PLATFORM)),)
-  export PLATFORM_DIR = ../../$(PLATFORM)
-else
-  $(error [ERROR][FLOW] Platform '$(PLATFORM)' not found)
-endif
+#-------------------------------------------------------------------------------
+# Setup varialbles to point to root / head of the OpenROAD directory 
+# - the following settings allowed user to point OpenRoad binaries to different
+#   location
+# - default is current install / clone directory
+export OPENROAD_HOME ?= .
```

Comment:
Since this is the flow home and not where the openroad app is located, wouldn't this variable be more descriptive as FLOW_HOME?